b2/console_tool.py

```diff
@@ -1040,6 +1042,53 @@ def _setup_logging(self, args, command, argv):
             logger.info('starting command [%s] with arguments: %s', command, argv)
 
 
+DATE_PATTERN = re.compile(r'^..., (\d\d) (...) (\d\d\d\d) (\d\d:\d\d\:\d\d) GMT$')
+
+MONTHS = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec']
+
+
+def clock_skew_hook(http_response):
+    """
+    Raises an exception if the clock in the server is too different from the
+    clock on the local host.
+
+    The Date header contains a string that looks like: "Fri, 16 Dec 2016 20:52:30 GMT".
+    The strptime function uses the current locale for month and day names like
+    "Dec" and "Fri", so we convert them to numbers ourselves.
+    """
+    # Make a string that uses month numbers instead of month names
+    server_date_str = http_response.headers['Date']
+    match = DATE_PATTERN.match(server_date_str)
+    if match is None:
+        raise DateFormatBad('date from server is: ' + server_date_str)
```

Comment:
the formatting of this message should not be here, but in `ClockSkew` class, the same as with other exceptions we have.